telemeta/templates/telemeta/base.html

```diff
@@ -83,12 +83,33 @@
   {{ user.username }}
 {% endif %}
 <a href="{% url "telemeta-flatpage" "help" %}">{% trans "Help" %}</a> |
-<a href="{% url "telemeta-logout" %}">{% trans "Sign out" %}
+<a href="{% url "telemeta-logout" %}">{% trans "Sign out" %} |
 <img src="{{ STATIC_URL }}telemeta/images/logout.png" alt="logout" style="vertical-align:middle" /></a>
 {% else %}
 <a href="{% url "telemeta-flatpage" "help" %}">{% trans "Help" %}</a> |
-<a href="{% url "telemeta-login" %}">{% trans "Sign in" %}</a>
+<a href="{% url "telemeta-login" %}">{% trans "Sign in" %}</a> |
 {% endif %}
+<form action="/i18n/setlang/" method="post" style="display:inline-block;">
```

Comment:
I'm not so happy with this because it is very intrusive in the design... Can you explain what is the use case for this stuff?
Remember that the language is defined by the browser automatically and can be changed in the profile page. If we need to setup anonymously, I think we can put it elsewhere.